bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -2500,4 +2500,121 @@ protected function processAvailableFieldDefinitions($fds, &$firstOne, &$commonFi
             }
         }
     }
+    
+    /**
+     * @Route("/export-json-config", methods={"GET"})
+     *
+     * @param Request $request
+     * 
+     * @return Response
+     */
+    public function exportJsonConfigAction(Request $request)
+    {
+        $classId = $request->get("classId");
+        $configData = json_decode($request->get('config'), true);
+        
+        try {
+            
+            $configName = $configData['shareSettings']['configName'];
+            
+            if(empty($configName)){
+                $configName = date("YmdHis")."_".$classId."_configuration";
+            }
+            
+            $jsonResponse = new JsonResponse(json_encode($configData), 200, [
+                'Content-Disposition' => 'attachment; filename="'.$configName.'.json"'
+            ], true);
+            
+            return $jsonResponse;
+            
+        } catch (\Exception $e) {
+            throw new \Exception("Error retrieving import configuration - ".$e->getMessage());
+        }
+    }
+    
+    /**
+     * @Route("/import-json-config", methods={"POST"})
+     *
+     * @param Request $request
+     * @param ImportService $importService
+     *
+     * @return JsonResponse
+     */
+    public function importJsonConfigAction(Request $request, ImportService $importService)
```

Comment:
maybe change this to
importCSVImportConfigFromJsonAction or something simpler (same for route).

Currently it is not clear what this is all about :-)

Reply:
route and action renamed ;)